mood-analysis.rb

```diff
@@ -19,6 +20,58 @@ def analyze_mood(words)
   return ":-|"
 end
 
+def strip_punctuation(words)
+  words.gsub(/[!.#,]/, "")
+end
+
+def happy_days(array)
+  entries = 0
+  happy_faces = 0
+  array.each do |line|
+    if happy_faces < 3
+      emotion = analyze_mood(line)
+      entries += 1
+      if emotion == ":-)"
+      happy_faces += 1
+      break if happy_faces == 3
+      end
+    end
+  end
+  if happy_faces == 3
+    return "It takes #{entries} entries for 3 happy days to occur"
+  else
+    return "Not enough happy days!"
+  end
+end
```

Comment:
I'd rather this method return just the fixnum of the number of days it takes, not a string with that information inserted inside it. This allows us the ability to do other things with this information instead of just meeting this specfic need.